js/ui/overlay/ui.overlay.js

```diff
@@ -1286,7 +1281,12 @@ var Overlay = Widget.inherit({
     _getContainer: function() {
         var position = this._position,
             container = this.option("container"),
-            positionOf = position ? position.of || window : null;
+            positionOf = null;
+
+        if(!container && position) {
+            var isEvent = !!(position.of && position.of.type === "positionEvent");
```

Comment:
> position.of.type === "positionEvent"

There's no such thing. You can adopt logic from [animation/position](https://github.com/DevExpress/DevExtreme/blob/19_2/js/animation/position.js#L328)